R/03-modelResults2DKernel.R

```diff
@@ -364,6 +364,9 @@ modelResults2DKernelUI <- function(id, title = "", asFruitsTab = FALSE){
           numericInput(inputId = ns("centerX"),
                        label = "Center point longitude",
                        min = -90, max = 90, value = c(), step = 0.5, width = "100%"),
+          numericInput(inputId = ns("decimalPlace"),
+                       label = "Input decimal places for map legend",
+                       min = 0, max = 10, value = 2, step = 1, width = "100%"),
```

Comment:
The name might be misleading. I first thought, the decimal places of the colour legend will change. Could you say:
"Decimal places for Mean/Error at the Center point" instead of 
"Input decimal places for map legend" ?

Reply:
Actually, I would like to see a small module here. You have to use this input in several maps, but copy/paste of code is always a source of bugs if one later wants to change something (e.g. the label, range etc.) It is better to have one place where we change the input and only call the module in all the maps. Lets talk about this. :slightly_smiling_face: